test/input/export_pagerank_delta_UW.gt

```diff
@@ -0,0 +1,72 @@
+% This file is copied from a repo - (https://github.com/bespoke-silicon-group/hb_starlite/tree/master/py-graphit-example)
```

Comment:
Similar to the above comment, I think this file is from our own pagerankdelta implementation? Or is this also from UW folks?

Reply:
this is from UW folks

Reply:
But I don't see the pagerankdelta program in the link in the comment? There's only sssp?